compiler/resolution/wrappers.cpp

```diff
@@ -2710,10 +2710,6 @@ static void initPromotionWrapper(PromotionInfo& promotion,
   retval->addFlag(FLAG_PROMOTION_WRAPPER);
   retval->addFlag(FLAG_FN_RETURNS_ITERATOR);
 
-  if (fn->hasFlag(FLAG_OPERATOR)) {
-    retval->addFlag(FLAG_OPERATOR);
-  }
-
```

Comment:
It'd be good to call this movement out in the PR message, but it definitely looks like an improvement